src/main/java/com/coreos/jetcd/EtcdLeaseImpl.java

```diff
@@ -77,7 +77,10 @@ public EtcdLeaseImpl(final ManagedChannel channel, EtcdLeaseHandler etcdLeaseHan
      * Init the request stream to etcd
      * start schedule to keep heartbeat to keep alive and remove dead leases
      */
-    public void startKeepAliveService(){
+    public void startKeepAliveService() throws LeaseKeepAliveServiceAlreadyStart{
```

Comment:
try to keep commit clean. for example, in this design doc commit, we should not change code.